src/Features/CSharp/Portable/CSharpFeaturesResources.resx

```diff
@@ -586,4 +586,7 @@
     <value>Introduce 'using' statement</value>
     <comment>{Locked="using"} "using" is a C# keyword and should not be localized.</comment>
   </data>
+  <data name="Make_ref_struct" xml:space="preserve">
+    <value>Make ref struct</value>
```

Comment:
needs a comment that both `ref` and `struct` are locked. (see above).  also, consider making this `Make 'ref struct'`